pkg/util/ovfcache/ovfcache.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package ovfcache
+
+import (
+	"context"
+	"time"
+
+	"github.com/vmware/govmomi/ovf"
+
+	"github.com/vmware-tanzu/vm-operator/pkg/util/ovfcache/internal"
+)
+
+// VersionedOVFEnvelope is an OVF envelope with an associated content version.
+type VersionedOVFEnvelope = internal.VersionedOVFEnvelope
+
+// GetterFn returns an OVF envelope for a provided library item ID.
+type GetterFn = internal.GetterFn
+
+const (
+	maxItems            = 100
+	expireAfter         = 30 * time.Minute
+	expireCheckInterval = 5 * time.Minute
+)
+
+// ErrNoGetter is returned from GetOVFEnvelope if there is no getter function.
+var ErrNoGetter = internal.ErrNoGetter
```

Comment:
Ditto.

Reply:
This may not be referenced anywhere at the moment, but we leave it public to ensure it can be compared if necessary.